app/views/requests/request_mailer/recap_in_library_confirmation.html.erb

```diff
@@ -0,0 +1,45 @@
+<tr class="email-header">
+  <td colspan="2" style="padding: 18px;border-top: 6px solid #e77500;">
+    <%= render 'request_header_patron' %>
+  </td>
+</tr>
+
+<tr>
+  <td colspan="2" style="padding: 18px;">
+    <p class="body-content" style="margin-bottom:0;">
+      <% if @submission.access_only? %>
+      <%= I18n.t('requests.recap_guest.email_conf_msg') %>
+      <% else %>
+      <%= I18n.t('requests.recap.email_conf_msg') %>
+      <% end %>
+    </p>
+  </td>
+</tr>
+<tr>
+  <td colspan="2" style="padding: 18px;">
+    <p class="body-content" style="margin-bottom:0;">
+      Once you recieve the confirmation that your book is available, please utilize our study-browse to make an appointment to come in and view the item in the Library.
```

Comment:
Rewrite as: ```Once you receive confirmation via email that your book is available, please utilize our study-browse service to make an appointment to come in to retrieve the item from the Library's circulation desk and view it in the Library.```

Reply:
That seems like a really long single sentence.  How about `Once you receive confirmation via email that your book is available, please utilize our study-browse service to make an appointment.  During your appointment you will retrieve the item from the Library's circulation desk and view it in the Library.`